handler.go

```diff
@@ -75,8 +80,12 @@ func (m *methods) callLoopEvent(c *Channel, event string) {
 	if !ok {
 		return
 	}
+	if len(errors) == 0 || !f.ArgsPresent {
```

Comment:
f.ArgsPresent is true only if our defined handler was declared receiving more than 1 param